Source/Models/SwishError.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import Argo
+
+public enum SwishError {
+  case SessionError(NSError)
+  case ServerError(code: Int, json: AnyObject)
+  case JSONSerializationError(NSError)
```

Comment:
It looks like this case covers both serialization and deserialization. Is there value in separating those out? My first impression of this was to wonder where the deserialization case was, before I read the code and realized it was included here.